components/ws-proxy/pkg/proxy/pass.go

```diff
@@ -128,6 +128,8 @@ func proxyPass(config *RouteHandlerConfig, infoProvider common.WorkspaceInfoProv
 			}
 			return
 		}
+		req = withResource(req, targetResource)
```

Comment:
:bubbles: I wonder if we could make it clear in these function's names that they are related to metrics. Because otherwise, I as a reader without context, would be confused, and need to look them up to understand what they do.

I don' have a good name right now, but what about `withResourceMetricLabel` or sth.

Reply:
That's a good suggestion, I had the same problem initially when reverse engineering it.

Reply:
Fixed in 785377d532fa858ef74453a2e9bbd009aeb0b7e4